.github/workflows/requirements.yaml

```diff
@@ -0,0 +1,22 @@
+name: requirements
+
+on: [pull_request, push]
+
+jobs:
+  requirements:
+
+    runs-on: ubuntu-latest
```

Comment:
Should we test other OSs?  I guess it depends on whether we are checking that the environment can be installed easily, or checking whether the package versions are compatible (which is a weaker requirement than the package versions can be installed easily on all OSs).

Reply:
Although I'm a little worried we will open a can of worms, it does sound useful to check on windows and osx. Can we give it a try?

Reply:
Or maybe just add the other OSs to the pytest workflow?